core/modules/path/path.admin.inc

```diff
@@ -643,9 +643,14 @@ function path_bulk_update_form($form, &$form_state) {
       // vocabularies.
       if (isset($path_info['pattern items'])) {
         foreach ($path_info['pattern items'] as $item => $item_label) {
+          $human_name = $item;
+          // Use the human name in the row title for vocabularies, not the machine name they are keyed by.
```

Comment:
This comment exceeds the 80 chars max limit. Also, since this is not a table in an end-user context, may I suggest changing it to something like this:

> For vocabularies, use their human name in the checkbox label instead of their machine name.

...and while at it, perhaps also move it inside the `if` conditional?

Reply:
@klonos Thanks for the feedback! I changed the comment to what you suggested and moved it inside the `if`.